javascript/react-js/inputs-and-lists.md

```diff
@@ -84,14 +86,14 @@ class App extends Component {
     const { task, tasks } = this.state;
 
     return (
-      <div className="col-6 mx-auto mt-5">
+      <div>
```

Comment:
Bootstrap's framework relies on a lot of divs to wrap elements. Removing the Bootstrap classes from these divs is a great start, but it may be a poor practice to keep all of these unnecessary divs in the HTML. I think we should remove these divs as well, since, for example, we really don't need a div wrapping a single button. The outermost div can stay for React's single root element needs.

Reply:
Hi, @twalton83 

Good catch! I am gonna make all the chances you have suggested. Thanks for the advice.